torchbox/torchbox.com
# Footer design review

[Link to Ticket](https://torchbox.monday.com/boards/1192293412/pulses/1405566963)

[Figma link](https://www.figma.com/file/QdMXBHr57mN1svsaNQ7Ggj?node-id=3934:53709&mode=design#707897480)

### Description of Changes Made
- Tightened spacing around components in footer
- Fixed image dimensions of footer logos
- Change background color and filter of footer logos

### How to Test
- Navigate to any page
- Ensure that the changes requested in the figma link provided above are implemented

### Screenshots
<details>
  <summary>Desktop</summary>
  <img width="2048" alt="image" src="https://github.com/torchbox/torchbox.com/assets/40872556/2b61aab3-28a8-4e2d-8010-b42ceb573881">
</details>
<details>
  <summary>Mobile</summary>
  <img width="457" alt="image" src="https://github.com/torchbox/torchbox.com/assets/40872556/b4d236d4-340d-40b9-ba85-6f870f3991d5">
</details>

### MR Checklist


#### Unit tests


#### Documentation


#### Browser testing

  - Latest version of Chrome on mac
  - Latest version of Firefox on mac
  - Latest version of Safari on mac
  - Safari on last two versions of iOS
  - Chrome on last two versions of Android

#### Data protection


#### Accessibility


#### Sustainability


#### Pattern library
